packages/python-neo4j-driver/PKGBUILD

```diff
@@ -0,0 +1,23 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+#
+# This PKGBUILD was extracted from AUR.
+# Creator: tuftedocelot tuftedocelot@fastmail.fm<
+# Changes by: petruknisme
+
+_pkgname=neo4j-python-driver
+pkgname=python-neo4j-driver
+pkgver=1.7.6
+pkgrel=1
+pkgdesc='Neo4j Bolt driver for Python'
+arch=('any')
+url='https://github.com/neo4j/neo4j-python-driver'
+license=('Apache')
+depends=('python' 'python-pytz')
+makedepends=('python-setuptools')
+source=("https://github.com/neo4j/$_pkgname/archive/$pkgver.zip")
+sha256sums=('00752aef0b36b54ec5badbebda68d8a0c9df3f8101aa2a49f5dbf975c59d1b4b')
+
+package() {
+  cd "$srcdir/$_pkgname-$pkgver"
```

Comment:
Use `cd $_pkgname-$pkgver`, there isn't need to call `$srcdir` or use quotes.

Also add newline between `cd *` line and `python *` line. Please see our [PKGBUILD templates](https://github.com/BlackArch/blackarch-pkgbuilds).

Reply:
I'm following PKGBUILD-python-lib template for `cd srcdir` https://github.com/BlackArch/blackarch-pkgbuilds/tree/master/PKGBUILD-python-lib

Reply:
Looks like it's _wrong_ there! But please follow my changes request. Also, please add a `build()` step, I forget that.

Reply:
Okay. Will do.